### PR TITLE
DDF-3347 updated instructions for changing port number

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates instructions for changing the HTTP and/or HTTPS port numbers.

#### Who is reviewing it? 

@mcalcote @vinamartin @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR. 
@lessarderic
@rzwiefel
@shaundmorris

#### How should this be tested?

review revised content.

#### What are the relevant tickets?
[DDF-3347](https://codice.atlassian.net/browse/DDF-3347)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_configuring/http-port.adoc [hunk 1]
```diff
@@ -1,17 +1,23 @@
-:title: Configuring HTTP Port
+:title: Configuring HTTP(s) Ports
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/http-port.adoc]: ❓ Should the `s` be capitalized here and in the summary?

File: distribution/docs/src/main/resources/content/_configuring/http-port.adoc [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
+:summary: Configure HTTP(s) Port.
 :order: 01
 
 == {title}
 
+To change HTTP or HTTPS ports from the default values, edit the `system.properties` file.
+
+. Open the file at ${home_directory}/etc/system.properties
+. Change the value after the `=` to the desired port number(s):
+.. `org.codice.ddf.system.httpsPort=8993` to `org.codice.ddf.system.httpsPort=<PORT>`
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/http-port.adoc]: ❓ Is there a alias for `org.codice.ddf` like there is for branding?

File: distribution/docs/src/main/resources/content/_configuring/http-port.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/http-port.adoc]: yes. fixed.

File: distribution/docs/src/main/resources/content/_configuring/http-port.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/http-port.adoc]: currently, no

File: distribution/docs/src/main/resources/content/_configuring/http-port.adoc [hunk 3]
```diff
@@ -1,17 +1,23 @@
-:title: Configuring HTTP Port
+:title: Configuring HTTP(S) Ports
 :type: configuration
 :status: published
 :parent: Configuring Federation
-:summary: Configure HTTP Port
+:summary: Configure HTTP(S) Port.
```
Comment (emanns95) [inline distribution/docs/src/main/resources/content/_configuring/http-port.adoc]: Small nit: Plural "ports" in title vs summary

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/http-port.adoc]: fixed.

**Comment by ricklarsen** [comment]
build now

**Pull request merged**